org.kde.plasmatube.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.kde.plasmatube",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "plasmatube",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        "flatpak/mpv.json",
+        "flatpak/python3-yt-dlp.json",
+        {
+            "name": "plasmatube",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/plasma-mobile/23.01.0/plasmatube-23.01.0.tar.xz",
+                    "sha256": "d23d2cb6c669a61f763e48aad44d02fdb141f81e4669ed59954ca9676a501c1a",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 326638,
```

Comment:
This points to to [kasts](https://release-monitoring.org/project/326638/). It looks like Plsmatube is currently not monitored by Anitya.